### PR TITLE
feat: harmonize colors on dashboard

### PR DESCRIPTION
**Pull request by martinvibes**
hello @djeck1432 please kindly review :)

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -406,54 +406,43 @@ def _get_risk_level(debt_to_supply_ratio: float) -> str:
...
         if not protocol:
             streamlit.write(f'Selected random protocol = {random_protocol}.')
             protocol = random_protocol
-
     loan = loans_data.loc[
         (loans_data['User'] == user)
         & (loans_data['Protocol'] == protocol),
     ]
-
-    if loan.empty:
```
Comment (djeck1432) [inline app.py:431]: Why did you remove this `if loan.empty`?

File: app.py [hunk 2]
```diff
@@ -16,6 +16,17 @@
 import src.settings
 import src.swap_amm
 
+# color map
+COLORS_MAP = {
```
Comment (djeck1432) [inline app.py:20]: You created this class and use only `Collateral` and `Debt` why do you need others key:value?

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Comment by martinvibes** [comment]
hello @djeck1432 i have implemented requested changes 
Kindly review :)

**Comment by djeck1432** [comment]
Screenshot 2024-10-07 at 12 07 47
Hi, while running your code, I got this error, please fix that

**Pull request closed without merging**